### PR TITLE
Add more shared logic

### PR DESCRIPTION
**Pull request by SUPERCILEX**

### CODE REVIEW

File: src/test/groovy/com/auth0/gradle/oss/AndroidLibraryPluginTest.groovy [hunk 1]
```diff
@@ -5,12 +5,19 @@ import org.gradle.testfixtures.ProjectBuilder
...
+import java.nio.file.Files
+
 class AndroidLibraryPluginTest extends Specification {
 
     Project project
 
     void setup() {
         project = ProjectBuilder.builder().build()
+        Files.copy(
+                new File("local.properties").toPath(),
```
Comment (lbalmaceda) [inline src/test/groovy/com/auth0/gradle/oss/AndroidLibraryPluginTest.groovy]: Why is this required?

**Comment by SUPERCILEX** [comment]
> So in terms of "migration notes", those previously adding the lifecycle-plugin, extending the clean task, and configuring jacoco on their projects will now need to stop, as that's going to be done by this plugin. Correct?

I think it will technically still work to configure that stuff, but yeah, there's no point in doing it anymore.

File: src/test/groovy/com/auth0/gradle/oss/AndroidLibraryPluginTest.groovy (hunk 1 above)
Comment (SUPERCILEX) [reply src/test/groovy/com/auth0/gradle/oss/AndroidLibraryPluginTest.groovy]: Because the Android Gradle Plugin needs to know where the Android SDK is. These tests are broken on master.

**Comment by lbalmaceda** [comment]
Thank you. I'll schedule a release for the coming weeks
